py4cast/io/writing_outputs.py

```diff
@@ -0,0 +1,377 @@
+import os
+from copy import deepcopy
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from cfgrib import xarray_to_grib as xtg
+from dataclasses_json import dataclass_json
+
+from py4cast.datasets.base import DatasetABC, NamedTensor
+from py4cast.forcingutils import compute_hours_of_day
+
+
+@dataclass_json
+@dataclass
+class GribSavingSettings:
+    template_grib: str
+    directory: str
```

Comment:
consider using **pathlib.Path** for wqorking with file and dir path.

Reply:
pathlib.Path deserialization is a mess with dataclass json 
https://github.com/marshmallow-code/marshmallow/issues/1815
that would require writing a wrapper for path objects. Not difficult, but I don't want to fiddle with this since the scope is so narrow. If you have a suggestion on how to do it with no boilerplate, I gladly accept, otherwise let's skip that.